parts/configure-swarmmode-cluster.sh

```diff
@@ -131,7 +131,17 @@ echo "Installing and configuring Docker"
 installDocker()
 {
   for i in {1..10}; do
-    wget --tries 4 --retry-connrefused --waitretry=15 -qO- https://get.docker.com | sh
+    echo "Adding Docker’s public key for CS packages..."

```

Comment:
Curious about your reasons to add steps individually instead of using get.docker.com? Was this for installing additional kernel modules? Fyi these will change anyways we add Azure mirrors for all docker files.